Amjith/release 1.6.0

amjith opened the pull request:
This is in preparation for 1.6.0 release. 

Can the @dbcli/mycli-core  team please review the change log to make sure I didn't skip something? 

I'll merge this in once I get a :+1: from a majority of core devs.

tsroten commented:
@amjith Thanks for preparing this release. I think the changelog is missing #219 and #226.

matheus-iclinic commented on changelog.md:
```diff
@@ -1,3 +1,35 @@
+Features:
+---------
+
+* Change continutation prompt for multi-line mode to match default mysql. 
```
@amjith Here you mean "continuation", right? :smile:

mdsrosa replied:
@amjith Here you mean "continuation", right? :smile:

amjith commented:
I'm sorry I missed the two features @tsroten, it was an oversight. I've fixed it now. 

The typo is also fixed @mdsrosa. 

Can you take another look?

tsroten commented:
@amjith Looks good to me!

mdsrosa commented:
@amjith Looks good to me too!

amjith commented:
I'm going to merge this and do a release. We'll release the stdin feature in the next release. I'm planning to do more frequent releases (once every couple of weeks).
